packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -608,4 +617,55 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 		provider.opProcessingController.resumeProcessing();
 		await uploadP;
 	});
+
+	it("reconnection does not block ops when having pending blobs", async () => {
+		const container1 = await provider.makeTestContainer(testContainerConfig);
+		const dataStore1 = await requestFluidObject<ITestDataObject>(container1, "default");
+		const blobManager = (container1 as any).runtime.blobManager;
```

Comment:
this seems like the wrong layer to mock. we want to ensure the blob manager work correctly, so i think we should be mocking the driver call